### ISSUE TITLE
Co-op

### ISSUE THREAD

**Issue by Nasdneb** [wizardamigosinstitute/organisation#62]
Which legal forms to choose for our business

**Comment by serapath** [comment]
Formally, I would go with `"Freelancer"` or Single Person Companies (e.g. `Ltd.`) and `Contracts` between them.

What kind of contracts, thus who has which rights and duties, maybe we can brainstorm [here](https://github.com/digitalsolopreneur/company/blob/master/CONCEPT.md), just open an **issue** [here](https://github.com/digitalsolopreneur/company/issues)

**Comment by Nasdneb** [comment]
- Reputation Points
- point Creation Rate
- point Distribution Rate
- redistribution Pool (internal coins)
- Money comes in -> Goes into group fund (external money)
- group fund will pay for company costs (rent, domain, server etc)
- rest of group fund will be distributed to community (respecting points)
- vote using your rep. points

Join and leave?
Kickstarter (support poins)
When payments - how to distribute based on reputation points.

-> Write simple server application with simple web frontend
 -> Then we can figure out the tricky parts

**Comment by serapath** [comment]
Do you have a repository for the code or bot already?
Maybe you can push it to github while you work on it, so others can take a look and maybe even contribute :-) ... if it's written in nodejs, maybe the work can even be split into modules so others can contribute some of the necessary modules - if they don't exist on npm yet.

**Comment by serapath** [comment]
There are a lot of `npm modules` that make working with the github API easier

**Comment by Nasdneb** [comment]
yup the repo is at https://github.com/wizardamigosinstitute/RepPointsServer

I guess it's easiest if we develop it in node.js, coz then everyone can contribute

**Comment by Nasdneb** [comment]
Hey Alex did you check a GbR for the wizard school? Doesn't cost anything, no notary services needed, but the members are liable with their own money

**Comment by serapath** [comment]
yes, that's the crap - they are liable personally and you cannot buy or sell shares so easily.
Maybe we can check more in detail, otherwise we could try with an english "Ltd." or maybe estonia has an alternative option.
I started to summarize it here: https://github.com/digitalsolopreneur/company/blob/master/COMPANY.process.md
